### ISSUE TITLE
Flickering and selection issues (possibly related to overlay)

### ISSUE THREAD

**Issue by humanbeinc** [jackocnr/intl-tel-input#896]
### See bug in action
https://static.drsam.de/web/intl-tel-input-master/bug_intl_tel.webm
https://dr-sam.webflow.io/

### Steps to reproduce
1.  Just init the plugin with the steps below

### Expected behaviour
normal behaviour

### Actual behaviour
Flickers and countryselection is wrong or not working

### Initialisation options
```
<script src="https://static.drsam.de/web/intl-tel-input-master/build/js/intlTelInput.min.js"></script>
<script type="text/javascript">
var callformvsPhoneInput = document.querySelector("#phone-2");
window.intlTelInput(callformvsPhoneInput, {
  initialCountry: "auto",
  nationalMode: true,
  autoPlaceholder: "aggressive",
  preferredCountries: ["de", "ch", "at"],
  utilsScript: "https://static.drsam.de/web/intl-tel-input-master/build/js/utils.js" // just for formatting/placeholders etc
});
</script>
```

**Comment by jackocnr** [comment]
Wow that's really weird.

If you can find the minimal set of code that reproduces the problem and put that in a CodePen, then I'd be happy to take a look. Here is [an example pen](https://codepen.io/jackocnr/pen/yehvE) with the plugin up and running - feel free to fork this if it helps.

**Comment by jackocnr** [comment]
Closing due to inactivity.

**Comment by humanbeinc** [comment]
@jackocnr Auto-closing issues is false advertising for the stability/quality of this project, just saying.

**Comment by jackocnr** [comment]
As I made clear before: it's very unlikely to be an issue with the plugin. Nonetheless, I'd be happy to take a look if you can find the minimal set of code that reproduces the problem and put that in a CodePen.

**Comment by humanbeinc** [comment]
Found the issue: https://codepen.io/anon/pen/GaebYg

My template used a .highlight class with absolute positioning therefor it's not exactly a bug in the plugin but still a problem when otherwise-unrelated-mainstream-words are used as classnames...

**Comment by jackocnr** [comment]
> but still a problem when otherwise-unrelated-mainstream-words are used as classnames

Yeah I agree. And thank you for taking the time to narrow down the issue.

I think the best thing to do is to rename the plugin's CSS classes to use BEM e.g. call the root class something simple and terse like `.iti` and then everything else would be `.iti__selected-flag`. This is what the popular lib [pickadate.js](https://github.com/amsul/pickadate.js/blob/master/lib/themes/classic.css) does.

I'm afraid I don't have time to implement this myself at the moment, but would welcome a pull request.

**Comment by jackocnr** [comment]
This is now merged into master if you want to give it a spin - let me know if you find any issues. After a bit of testing I'll do a major release.

**Comment by jackocnr** [comment]
Released in v16.0.0.
